src/makeInfinityAdder.js

```diff
@@ -7,6 +7,23 @@
 
 function makeInfinityAdder() {
   // write code here
+  let res = 0;
+  let sum = 0;
+
+  const adder = (param) => {
+    if (param >= 0) {
```

Comment:
actually, not a perfect solution, because if your parameter is -5, for example, your function will not work correctly. In your task, you do not have a condition that your parameters will always be greater than 0.